nova/exception.py

```diff
@@ -1841,6 +1841,11 @@ class RequestedVRamTooHigh(NovaException):
                 "than the maximum allowed by flavor %(max_vram)d.")
 
 
+class RequestedRamSizeUnsupported(NovaException):
+    msg_fmt = _("The requested amount of memory %(req_ram)d is not supported "
+                "memory size.")
```

Comment:
I think there's a way to provide additional information into the exception message. The current way is too generic for me. I'd like to have the "not a multiple of 4" in there. There are other exceptions that allow the raising code to pass an additional "reason" iirc.